content/en/docs/Deployment/kubernetes-deploy.md

```diff
@@ -142,14 +144,20 @@ controllers to watch for CRD´s instances into the `kubecf` namespace.
 > If the `kubecf` namespace is deleted, but the operators are still running, they will no longer
 > know which namespace to watch. This can lead to problems, so make sure you also delete the pods
 > inside the `cfo` namespace, after deleting the `kubecf` namespace.
+{{% /alert %}}
+
+{{% alert title="Also" %}}
```

Comment:
```suggestion
{{% alert title="Note" %}}
```
Having a section titled `Also` is kind of odd; just repeating it instead should be fine (given that it's just a callout, not a section header in the ToC).